valhalla/baldr/nodeinfo.h

```diff
@@ -389,10 +385,12 @@ class NodeInfo {
 protected:
   // Organized into 8-byte words so structure will align to 8 byte boundaries.
 
-  // 26 bits for lat,lon offset allows 7 digit precision within 4 degree tiles. Note that
-  // this would require using double precision to actually achieve this precision.
-  uint64_t lat_offset_ : 26; // Latitude offset from tile base latitude
-  uint64_t lon_offset_ : 26; // Longitude offset from tile base longitude
+  // 26 bits for lat,lon offset allows 7 digits of precision even in 4 degree tiles
+  // to stay backwards compatible we have to break 6 digits and the 7th digit into two parts
+  uint64_t lat_offset_ : 22; // Latitude offset from tile base latitude in int 6 digit precision
+  uint64_t lat_offset7_ : 4; // Latitude offset 7th digit of precision
+  uint64_t lon_offset_ : 22; // Longitude offset from tile base longitude in int 6 digit precision
+  uint64_t lon_offset7_ : 4; // Longitude offset 7th digit of precision
```

Comment:
this was a nice little trick i worked out to add a full 7 digits of precision to the nodes in the graph in a backwards compatible way. since previously we only did 6, it meant we only ever used 22 bit wide numbers at the most. since valhalla tiles assume little endian, i was able to just steal back the 4 unused bits and use them to store the 7th digit of precision.